playbooks/webapp/README.md

```diff
@@ -99,7 +99,7 @@ To delete the webapp:
 * **image_filter** (str): Name of AWS AMI to use. Default: `Fedora-Cloud-Base-35-*`
 * **sshkey_pair_name** (str): Name for the EC2 key pair. Default: `"{{ resource_prefix }}-key"`
 * **bastion_host_name** (str): Name for the EC2 instance. Default: `"{{ resource_prefix }}-bastion"`
-* **bastion_host_type** (str): Instance type for the EC2 instance. Default: `t2.xlarge`
+* **bastion_host_type** (str): Instance type for the EC2 instance. Default: `t3.micro`
```

Comment:
Is this also a breaking change?

Reply:
Yes, it does not break the code, but supposing that someone used the role in the past without setting the bastion_host_type, he is still expecting to have t2.xlarge and not t3.micro as default value.

Reply:
Retaining the old value and using t3.micro in the tests makes sense.